setup.py

```diff
@@ -66,7 +66,7 @@ def find_version():
     maintainer_email="greg@reinbach.com",
     license="BSD",
     packages=find_packages(include=["djmoney", "djmoney.*"]),
-    install_requires=["setuptools", "Django>=2.2", "py-moneyed>=1.2,<2.0"],
+    install_requires=["setuptools", "Django>=2.2", "py-moneyed>=2.0"],
```

Comment:
Maybe here we can be conservative to the next major `py-moneyed` version ?
```suggestion
    install_requires=["setuptools", "Django>=2.2", "py-moneyed>=2.0,<3.0"],
```

Reply:
In my experience that isn't common practice for package dependencies and might cause more trouble than its worth, i.e. it will put users in impossible situations if they want to upgrade py-moneyed before there is an explicitly compatible version of django-money.

Just two quick examples:

- https://github.com/samuelcolvin/pydantic/blob/master/setup.py#L136
- https://github.com/django/django/blob/main/setup.cfg#L40

Reply:
If we focus on pymoneyed, isn't the experience that we *should* in fact pin it below the next major version? Since we are talking django-money, this is a transient dependency that might make upgrade paths more difficult if it doesn't have a roof.